client/hosting/overview/components/plan-storage-bar.tsx

```diff
@@ -39,7 +48,10 @@ const PlanStorageBar: FC< PropsWithChildren< Props > > = ( { children, mediaStor
 				</span>
 			</div>
 
-			<ProgressBar color="var(--studio-red-30)" value={ percent } total={ 100 } compact={ false } />
+			<div className={ classes }>
```

Comment:
```suggestion
			<div className={ clsx( 'plan-storage__bar', {
				'is-alert': usagePercent > ALERT_PERCENT,
			} ) }>
```
Nit, but for future reference, let's default to inlining `clsx` calls directly in the `className` attribute. That's more conventional.